src/topp/IDConflictResolver.cpp

```diff
@@ -102,34 +102,52 @@ class TOPPIDConflictResolver :
   {
   }
 
+protected:
+
   void registerOptionsAndFlags_() override
   {
     registerInputFile_("in", "<file>", "", "Input file (data annotated with identifications)");
     setValidFormats_("in", ListUtils::create<String>("featureXML,consensusXML"));
     registerOutputFile_("out", "<file>", "", "Output file (data with one peptide identification per feature)");
     setValidFormats_("out", ListUtils::create<String>("featureXML,consensusXML"));
+    registerFlag_("make_unique", "A map may contain multiple features with both identical sequence and charge state. The feature with the highest intensity is very likely the most reliable one. When switched on, the filter removes the sequence annotation from the lower intensity features, thereby resolving the multiplicity. Only the most reliable features maintain annotated.");
```

Comment:
Apart from these minor issues, overall, it looks pretty good and I think we are close to a merge. Thanks!

One might argue that the feature with lowest m/z deviation to the ID or lowest RT deviation might be better and it is hard to reason why one is better than the other. Proper resolving this might be much more complex than just taking the highest intensity one (e.g., by formulating a global optimization problem). Reserving the name make_unique is, thus, way too general to assume that we will not change it in the future (and which potentially will break many workflows). So I am a bit hesitant.

Maybe naming the parameter -resolve_features "highest_intensity" would lead to more stable tool interface as we could add more algorithms in the future. The name "feature" implies that we are actually resolving IDs of ambiguous features (modified sequence + charge) and not peptides.

Reply:
I would also change it to "... maintain annotated with that peptide". It could still be annotated by other peptides.
And I agree that make_unique is too general. But I also don't like "resolve_features" because that actually sounds like you would always end up with only one annotation to a feature but it does it the other way: it makes the features unique to a peptidoform. 
I think there need to be two categories, one for resolution across peptides and one for resolution inside a feature.
One could argue, that some approach will try to optimize for both at the same time and probably also the order of the resolution will make a big difference, but it would be a first step.

Reply:
I think the order could be:
In feature:
- removing labeled ids that map to wrong channel (aka multiplex resolution)
- removing all lower scoring ones (standard)
- remove same final ids from features - except for the highest intensity one

Reply:
I still think the description needs a "... maintain annotated with that peptide"